services/brig/src/Brig/App.hs

```diff
@@ -157,7 +158,7 @@ import Wire.API.User.Identity (Email)
 import Wire.API.User.Profile (Locale)
 
 schemaVersion :: Int32
-schemaVersion = 79
```

Comment:
Was this one wrong? Because there's a migration with version 80.

Reply:
I just hope there won't be a problem in any installation due to this, as in a wrong thing was in place and it worked, but this made it right, but actually broke it.